### PR TITLE
[Tutorials] Continue moving metadata to frontmatter

### LINKED ISSUE #1281: [Docs RFC] Add keywords and tags to frontmatter of all pages
# Describe change in content, appearance, or functionality

Necessary for decoupling our related content and keywords from the page-index and eventually deprecating the index.

# Subject matter expert (SME)

[If known, who is a good person to ask about this topic]

# Deadline

[When does this need to be addressed]

# Any further info

[Anything else you want to add, or further links]

### PR DESCRIPTION
**Pull request by charislam**
# Description

[Short summary of why you created this PR]

# Links

Fixes #1281 

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Are procedure and highlight tags used appropriately?
- [ ] Has the index been updated appropriately?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] Are all links provided in reference style, and resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

**Review by Rajakavitha1 — APPROVED** [review]

File: timescaledb/tutorials/analyze-bitcoin-blockchain/ingest-query-btc-transactions.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+---
+title: Insert and query Bitcoin transactions
+excerpt: Ingest and store Bitcoin blockchain dataa in your database
```
Comment (Loquacity) [inline timescaledb/tutorials/analyze-bitcoin-blockchain/ingest-query-btc-transactions.md:3]:
```suggestion
excerpt: Ingest and store Bitcoin blockchain data in your database
```

File: timescaledb/tutorials/analyze-intraday-stocks/explore-stocks-data.md [hunk 2]
```diff
@@ -290,13 +297,16 @@ Change `date` to see the candlesticks for another day.
...
 * [Pandas docs][pandas-docs]
 * [Plotly docs][plotly-docs]
-* [Timescale free trial](https://www.timescale.com/timescale-signup)
-* [Analyze cryptocurrency data](/tutorials/analyze-cryptocurrency-data/)
-
+* [Timescale free trial][timescale-signup]
+* [Analyze cryptocurrency data][analyze-cryptocurrency-data]
 
+[alpha-vantage-api]: https://www.alphavantage.co/documentation/
+[analyze-cryptocurrency-data]: /timescaledb/:currentVersion:/tutorials/analyze-cryptocurrency-data/
```
Comment (Loquacity) [inline timescaledb/tutorials/analyze-intraday-stocks/explore-stocks-data.md:308]: not sure why Vale is complaining here, do we have a non-printable space or something? 🤔

File: timescaledb/tutorials/analyze-nft-data/nft-schema-ingestion.md [hunk 3]
```diff
@@ -1,3 +1,10 @@
+---
+title: NFT schema design and ingestion
+excerpt: Design a schema to ingset and store NFT transaction data
```
Comment (Loquacity) [inline timescaledb/tutorials/analyze-nft-data/nft-schema-ingestion.md:3]:
```suggestion
excerpt: Design a schema to ingest and store NFT transaction data
```

File: timescaledb/tutorials/aws-lambda/continuous-deployment.md [hunk 4]
```diff
@@ -99,8 +105,7 @@ Connect the Github repository to AWS Lambda using Github actions.
 
 ### Adding your AWS credentials to the repository
 You need to add your AWS credentials to the repository so it has permission to connect to Lambda.
-You can do this by adding [GitHub secrets](https://docs.github.com/en/actions/reference/encrypted-secrets)
-using the GitHub CLI.
+You can do this by adding [GitHub secrets][github-secrets] using the GitHub CLI.
```
Comment (Loquacity) [inline timescaledb/tutorials/aws-lambda/continuous-deployment.md:108]:
```suggestion
You can do this by adding [GitHub secrets][github-secrets] using the GitHub command-line.
```

File: timescaledb/tutorials/aws-lambda/index.md [hunk 5]
```diff
@@ -1,11 +1,17 @@
+---
+title: TimescaleDB with AWS Lambda
+excerpt: Learn to work with AWS Lambda and TimescaleDB together
```
Comment (Loquacity) [inline timescaledb/tutorials/aws-lambda/index.md:3]:
```suggestion
excerpt: Learn to use AWS Lambda and TimescaleDB together
```

File: timescaledb/tutorials/custom-timescaledb-dashboards.md [hunk 6]
```diff
@@ -1,3 +1,9 @@
+---
+title: Custom TimescaleDB dashboards
+excerpt: Build custom TimescaleD bdashboards with Hasura GraphQL and React
```
Comment (Loquacity) [inline timescaledb/tutorials/custom-timescaledb-dashboards.md:3]:
```suggestion
excerpt: Build custom TimescaleDB dashboards with Hasura GraphQL and React
```

File: timescaledb/tutorials/financial-candlestick-tick-data/query-candlestick-views.md [hunk 7]
```diff
@@ -1,3 +1,10 @@
+---
```
Comment (Loquacity) [inline timescaledb/tutorials/financial-candlestick-tick-data/query-candlestick-views.md:1]: Go home, Vale. You're drunk.

File: timescaledb/tutorials/grafana/geospatial-dashboards.md [hunk 8]
```diff
@@ -1,3 +1,9 @@
+---
+title: Use Grafana to visualize geospatial data stored in TimescaleDB
+excerpt: Use the WorldMap visualization to see geospatial data overload on a map of the world
```
Comment (Loquacity) [inline timescaledb/tutorials/grafana/geospatial-dashboards.md:3]:
```suggestion
excerpt: Use the WorldMap visualization to see a geospatial data overload on a map of the world
```

File: timescaledb/tutorials/grafana/installation.md [hunk 9]
```diff
@@ -1,3 +1,9 @@
+---
+title: Set up TimescaleDB and Grafana
+excerpt: Set up Managed Service for TimescaleDB with Grafana to visualize your data
```
Comment (Loquacity) [inline timescaledb/tutorials/grafana/installation.md:3]:
```suggestion
excerpt: Use Grafana to visualize your data on Managed Service for TimescaleDB
```

File: timescaledb/tutorials/grafana/installation.md [hunk 10]
```diff
@@ -1,3 +1,9 @@
...
+keywords: [Grafana, visualizations, analytics]
```
Comment (Loquacity) [inline timescaledb/tutorials/grafana/installation.md:4]:
```suggestion
keywords: [Grafana, visualization, analytics]
```

File: timescaledb/tutorials/grafana/setup-alerts.md [hunk 11]
```diff
@@ -1,3 +1,9 @@
+---
+title: Set up Grafana alerts
+excerpt: Get alerted when a problem occurs
```
Comment (Loquacity) [inline timescaledb/tutorials/grafana/setup-alerts.md:3]:
```suggestion
excerpt: Use Grafana to get alerted when a problem occurs
```

File: timescaledb/tutorials/grafana/visualizations/bar-chart.md [hunk 12]
```diff
@@ -1,3 +1,10 @@
+---
+title: Build a bar chart in Grafana
+excerpt: Plot a bar chart in Grafana to compare values between categories
+keywords: [Grafana, visualizations, analytics]
```
Comment (Loquacity) [inline timescaledb/tutorials/grafana/visualizations/bar-chart.md:4]:
```suggestion
keywords: [Grafana, visualization, analytics]
```

File: timescaledb/tutorials/grafana/visualizations/candlestick.md [hunk 13]
```diff
@@ -1,3 +1,10 @@
+---
+title: Build a candlestick chart in Grafana
+excerpt: Create a candlestick chart in Grafana to visualize opening, closing, high, and low prices of financial assets
+keywords: [Grafana, visualizations, analytics, finance]
```
Comment (Loquacity) [inline timescaledb/tutorials/grafana/visualizations/candlestick.md:4]:
```suggestion
keywords: [Grafana, visualization, analytics, finance]
```

File: timescaledb/tutorials/grafana/visualizations/histograms.md [hunk 14]
```diff
@@ -1,3 +1,10 @@
+---
+title: Build a histogram in Grafana
+excerpt: Create a histogram in Grafana to visualize the distribution of data
+keywords: [Grafana, visualizations, analytics]
```
Comment (Loquacity) [inline timescaledb/tutorials/grafana/visualizations/histograms.md:4]:
```suggestion
keywords: [Grafana, visualization, analytics]
```

File: timescaledb/tutorials/grafana/visualizations/index.md [hunk 15]
```diff
@@ -1,3 +1,9 @@
+---
+title: Grafana visualizations
+excerpt: Learn how to create Grafana visualizations of time-series data
+keywords: [Grafana, visualizations, analytics]
```
Comment (Loquacity) [inline timescaledb/tutorials/grafana/visualizations/index.md:4]:
```suggestion
keywords: [Grafana, visualization, analytics]
```

File: timescaledb/tutorials/grafana/visualizations/time-series.md [hunk 16]
```diff
@@ -225,7 +232,6 @@ change to your query.
 
 </procedure>
 
-
-[install-timescale]: /install/:currentVersion:/
 [gsg-data]: /timescaledb/:currentVersion:/getting-started/
+[install-timescale]: /install/latest/
```
Comment (Loquacity) [inline timescaledb/tutorials/grafana/visualizations/time-series.md:236]:
```suggestion
[install-timescale]: /install/:currentVersion:/
```

File: timescaledb/tutorials/monitor-mst-with-prometheus.md [hunk 17]
```diff
@@ -1,3 +1,10 @@
+---
+title: How to set up a Prometheus endpoint for Managed Service for TimescaleDB
+excerpt: Monitor your TimescaleDB service with Prometheus
```
Comment (Loquacity) [inline timescaledb/tutorials/monitor-mst-with-prometheus.md:3]:
```suggestion
excerpt: Use Prometheus to monitor your Managed Service for TimescaleDB
```

File: timescaledb/tutorials/monitor-mst-with-prometheus.md [hunk 18]
```diff
@@ -1,3 +1,10 @@
...
+product: mst
+keywords: [prometheus, monitork]
```
Comment (Loquacity) [inline timescaledb/tutorials/monitor-mst-with-prometheus.md:5]:
```suggestion
keywords: [prometheus, monitor]
```

**Review by Loquacity — APPROVED** [review]

**Pull request merged**